docs/deploying.md

```diff
@@ -481,6 +481,25 @@ OpenStackDataPlaneNodeSet has the following status fields:
 | DNSClusterAddresses | |
 | CtlplaneSearchDomain | |
 
+##### OpenStackDataPlaneNodeSet Pods
+
+Pods within OpenStackDataPlaneNodeSet can fail for many reasons.
```

Comment:
There aren't pods in a nodeset, and I don't think that's the right way to think about the design conceptually. The pods this section are talking about are actually started by a dataplane Deployment by way of AnsibleEE by way of a Job, then a pod.

Reply:
Good point.